main/blinkConfig.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+var IDLFileContents = foam.lookup('org.chromium.webidl.IDLFileContents');
+var GitilesIDLFile = foam.lookup('org.chromium.webidl.GitilesIDLFile');
+var WebPlatformEngine = foam.lookup('org.chromium.webidl.WebPlatformEngine');
+var Parser = foam.lookup('org.chromium.webidl.Parser');
+
+var gitilesBaseURL = 'https://chromium.googlesource.com/chromium/src/+';
```

Comment:
This is only used once, like all of the other constants in this file. I think being consistent in either inlining or not is a good idea. I'd lean towards inlining personally.